relay-server/src/extractors/request_meta.rs

```diff
@@ -413,6 +409,21 @@ impl PartialMeta {
         if self.user_agent.is_some() {
             complete.user_agent = self.user_agent;
         }
+
+        if self.client_hints.sec_ch_ua_platform_version.is_some() {
+            complete.client_hints.sec_ch_ua_platform_version =
+                self.client_hints.sec_ch_ua_platform_version;
+        }
+        if self.client_hints.sec_ch_ua_platform.is_some() {
+            complete.client_hints.sec_ch_ua_platform = self.client_hints.sec_ch_ua_platform;
+        }
+        if self.client_hints.sec_ch_ua_model.is_some() {
+            complete.client_hints.sec_ch_ua_model = self.client_hints.sec_ch_ua_model;
+        }
+        if self.client_hints.sec_ch_ua.is_some() {
+            complete.client_hints.sec_ch_ua = self.client_hints.sec_ch_ua;
+        }
```

Comment:
I'm not sure what the alternative to this looks like, but I don't really like going through every client hint here. I think moving this responsibility to the client hint struct is a better choice.

Reply:
hmm the function itself makes use of partial moves which makes it harder to extract the logic to a different function